openmandriva/24.07

```diff
@@ -0,0 +1,14 @@
+NAME="OpenMandriva Lx"
+VERSION="24.07 (ROME) Rolling"
```

Comment:
[This ROME looks like a rolling-based distro](https://wiki.openmandriva.org/en/home), we should probably name this file `openmandriva` 🤔

Reply:
good catch, i just pushed the renamed file.